libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -532,8 +532,33 @@ TickType_t uxReadTimeOut_ticks = ipconfigDNS_RECEIVE_BLOCK_TIME_TICKS;
  as gethostbyname() may be called from different threads */
 BaseType_t xHasRandom = pdFALSE;
 TickType_t uxIdentifier = 0U;
+#if( ipconfigUSE_DNS_CACHE != 0 )
+	BaseType_t xLengthOk = pdFALSE;
+#endif
 
+	#if( ipconfigUSE_DNS_CACHE != 0 )
+	{
+	if( pcHostName != NULL )
```

Comment:
The spacing looks strange here, should it be adjusted to use 4 spaces rather than a tab?

Reply:
Yes, you are right, 2 tabs were missing. Thanks